plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -177,6 +177,7 @@ default CNode describeStructure(T instance, ConfigurationContext context)
                 attribute.setJsonSchema(true);
             }
             CNode value = attribute.describeStructure(instance, context);
+            System.out.println("This is " + value.asSequence());
```

Comment:
@timja here value isn't a sequence :(

Reply:
no because describeStructure only returns mappings

Reply:
Please use system loggers even for debugging purposes. Forgetting `System.out` code in a merged commit might be a major issue for users

Reply:
Yeah :+1: